pkg/querier/queryrange/limits_test.go

```diff
@@ -17,6 +16,8 @@ import (
 	"go.uber.org/atomic"
 	"gopkg.in/yaml.v2"
 
+	"github.com/grafana/loki/pkg/util/math"
```

Comment:
hmm this one is isolated from other packages below (I'm not adding a diff suggestion for you to just apply because the order would be wrong anyway)